content/people/index.md

```diff
@@ -600,7 +600,8 @@ people:
   affiliation: Stanford University
   team:
   - core
-- firstName: Ryan
+- key: ryan-n
```

Comment:
its ***probably*** fine, but im not in love with the hyphen being in there...  can we use his andrew id instead?
```suggestion
- key: rnayebi
```
once this PR is done, im going to create a GH issue to try to ensure that name collisions like this dont happen again, and one way to accomplish that is to use andrew IDs which are already unique.